packages/web-components/fast-components/src/tree-item/tree-item.styles.ts

```diff
@@ -156,7 +165,6 @@ export const treeItemStyles: FoundationElementTemplate<ElementStyles, TreeItemOp
     }
 
     .items {
-        display: none;
```

Comment:
These are being conditionally rendered depending on when the tree is expanded or whether we the conditional flag is set, so I've removed this line of code as it seems unnecessary.